src/metric/definition.go

```diff
@@ -49,11 +49,20 @@ var APIServerSpecs = definition.SpecGroups{
 				Type: sdkMetric.RATE,
 			},
 			{
-				Name: "apiserverCurrentInflightRequests",
+				Name: "apiserverCurrentInflightRequestsMutating",
 				ValueFunc: prometheus.FromValueWithOverriddenName(
 					"apiserver_current_inflight_requests",
-					"apiserverCurrentInflightRequests",
-					prometheus.IncludeOnlyLabelsFilter("request_kind"),
+					"apiserverCurrentInflightRequestsMutating",
+					prometheus.IncludeOnlyWhenLabelMatchFilter(map[string]string{"request_kind": "mutating"}),
```

Comment:
what would happen if another label is present?

Reply:
I added a test case [here](https://github.com/newrelic/nri-kubernetes/pull/507/files#diff-feb770271b232b58c3cdf2f08592a3df14a22d3d2d9bd94f86255699a8071e86R878). I'll be more exhaustive and a a specific case with multiple labels.

Reply:
Added more tests here ffc5a85dd29850f2d6f3aa1b1c75cf39fbb2993c.